examples/nnunet_example/README.md

```diff
@@ -0,0 +1,40 @@
+# NnUNetClient Example
+
+This example demonstrates how to use the NnUNetClient to train nnunet segmentation models in a federated setting.
+
+By default this example trains an nnunet model on the Task04_Hippocampus dataset from the Medical Segmentation Decathelon (MSD). However, any of the MSD datasets can be used by specifying them with the msd_dataset_name flag for the client. To run this example first create a config file for the server. An example config has been provided in this directory. The required keys for the config are:
+
+```yaml
+# Parameters that describe the server
+n_server_rounds: 1
+
+# Parameters that describe the clients
+n_clients: 1
+local_epochs: 1 # Or local_steps, one or the other must be chosen
+
+nnunet_config: 2d
+```
+
+The only additional parameter required by nnunet is nnunet_config which is one of the official nnunet configurations (2d, 3d_fullres, 3d_lowres, 3d_cascade_fullres)
+
+One may also add the following optional keys to the config yaml file
+
+```yaml
+# Optional config parameters
+nnunet_plans: /Path/to/nnunet/plans/json
```

Comment:
Is this a folder of nnunet plans or just a single one?

Reply:
just a single one, I will change default